src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -914,5 +925,16 @@ export class TransactionApi implements ITransactionApi {
       messageHash: args.messageHash,
     });
     await this.cacheService.deleteByKey(key);
+    await this._setInvalidationTimeForKey(key);
+  }
+
+  private async _setInvalidationTimeForKey(key: string): Promise<void> {
+    await this.cacheService.set(
+      new CacheDir(`invalidationTimeMs:${key}`, ''),
```

Comment:
Nit: I'd suggest creating a helper function to get the key as it is used in multiple places.

Reply:
I've moved this logic to the `CacheService` in [1d4da4c](https://github.com/safe-global/safe-client-gateway/pull/970/commits/1d4da4c0eabcd9f200cb0e9e813d8535421aabea) so it's now being used in 3 places (besides test classes). 

I don't have a strong opinion about it, I'm fine with creating an `utils.ts` for the data sources layer, extracting the string part to a constant somewhere, or letting it as it is.